http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmApiClientProvider.scala

```diff
@@ -66,6 +72,41 @@ trait WsmApiClientProvider[F[_]] {
 }
 
 class HttpWsmClientProvider[F[_]](baseWorkspaceManagerUrl: Uri)(implicit F: Async[F]) extends WsmApiClientProvider[F] {
+
+  override def getWorkspace(token: String, workspaceId: WorkspaceId, iamRole: IamRole = IamRole.READER)(implicit
+    ev: Ask[F, AppContext]
+  ): F[Option[WorkspaceDescription]] =
```

Comment:
this is really the only interesting bit of code review-wise. I essentially wrap the wsm client with a helper method to handle nulls and sanitize the data into the case class all existing service code is expecting to minimize the surface of impact.  

It seemed of little value and fairly prone to bugs to migrate existing code to use the model the wsm client returns.

Reply:
Could you add the above explanation as a  comment/ docstring into the function?